### PR TITLE
Limit max download size

### PR DESCRIPTION
**Pull request by carlosgjs**
Opportunistically adding settings.json and .flake8
Bumping deployment instance size

### CODE REVIEW

File: .flake8 [hunk 1]
```diff
@@ -0,0 +1,2 @@
+[flake8]
+max-line-length = 120
```
Comment (lsetiawan) [inline .flake8:2]: Any reason why this max length is long?

**Review by lsetiawan — COMMENTED** [review]
Looks good as is, just wondering about the max line length being so long

File: .flake8 (hunk 1 above)
Comment (carlosgjs) [reply .flake8:2]: That's what I typically use. I find that with today's screens 80 is too limiting.

Comment (lsetiawan) [reply .flake8:2]: Gotcha! Okay. That sounds good.

**Pull request merged**